supporting_content/dimensions/chargefrequency.md

```diff
@@ -57,10 +55,11 @@ Current values observed in billing data for various scenarios:
 
 ## Clarifications
 
-- "Usage-Based" value means that charges will be incurred when usage occurs. If there's a charge that happens monthly, even if it's based on usage, it would have a frequency 'recurring' as it's occurrence does not depend on usage consumption but other contractual terms. 
-- An upfront RI is a cost incurred in one billing period, similar to EC2 usage. Although neither will recur into another billing period, the upfront RI will be a 'one-time' charge and EC2 usage will be 'usage-based'. Another separate and distinct upfront RI or record of EC2 usage can be charged in the next period, if they occur. If a upfront RI is bought every month, they will be individual 'one-time' charges and cannot be considered as 'recurring'. 
+- "Usage-Based" value means that charges will be incurred when usage occurs. If there's a charge that happens monthly, even if it's based on usage, it would have a frequency 'recurring' as it's occurrence does not depend on usage consumption but other contractual terms.
+- An upfront RI is a cost incurred in one billing period, similar to EC2 usage. Although neither will recur into another billing period, the upfront RI will be a 'one-time' charge and EC2 usage will be 'usage-based'. Another separate and distinct upfront RI or row of EC2 usage can be charged in the next period, if they occur. If a upfront RI is bought every month, they will be individual 'one-time' charges and cannot be considered as 'recurring'.
```

Comment:
This is the only change in this file. The formatting changes were auto-applied on save.